CHANGELOG.md

```diff
@@ -9,6 +9,11 @@ More detailed release notes can be found on the [releases page](https://github.c
 
 * [fyne-cli] Add support for passing custom build tags (#1538)
 
+### Changed
+
+* Run validation on content change instead of on each Refresh in widget.Entry
+* Run onValidationChanged() when calling entry.SetValidationError()
```

Comment:
I don't think we should list internal details in a changelog.

Reply:
Good point. It also kind of ties in with it not working correctly (in widget.Form that is) :)